### DISCUSSION TITLE
Notes - href Attribute support

### DISCUSSION THREAD

**Discussion by Minocula** [Kovah/LinkAce#507] [Ideas & Feature Requests]
I use LinkAce alot and its one of my primary used tools. So if I create / add a new Website record it is not finished there. Over the time I add details like notes. Often I add links in the notes of subpages of the current website / record. 

I request a function which does automatically create HTML Links also in the note section. I prefer target blank Attribute as the other links.

Example:
![2022-07-13 10_33_54-Minocula's link collection](https://user-images.githubusercontent.com/63090608/178689335-afecc004-0ac2-4566-b0e9-1f50f8861cc1.png)

**Comment by Minocula** [comment]
is there a chance to use this function in Linkace 2? Is the implementation so demanding? Does this perhaps help further?

https://stackoverflow.com/questions/49634850/convert-plain-text-links-to-clickable-links

**Reply by Kovah** [reply]
I think this could be an issue with the configuration. It can be tested in the demo:

- open https://demo.linkace.org/ and login with the demo account
- make sure the `Enable Markdown for descriptions and notes` option in the user settings is enabled
- open any link and paste the following into the note field: `Hello link test: https://github.com/Kovah/LinkAce/discussions/507` and save
- the note will be converted to Markdown with automatic links
